builtin/credential/ldap/path_config.go

```diff
@@ -48,6 +48,12 @@ func pathConfig(b *backend) *framework.Path {
 
 	tokenutil.AddTokenFields(p.Fields)
 	p.Fields["token_policies"].Description += ". This will apply to all tokens generated by this auth method, in addition to any configured for specific users/groups."
+
+	p.Fields["password_policy"] = &framework.FieldSchema{
```

Comment:
We could add `password_policy` to `ConfigEntry.PasswordlessMap()` so that it's returned in reads?